modules/Kconfig.cmsis_nn

```
@@ -0,0 +1,71 @@
+# Copyright (c) 2021, Commonwealth Scientific and Industrial Research
+# Organisation (CSIRO) ABN 41 687 119 230.
+# SPDX-License-Identifier: Apache-2.0
+
+comment "Components"
```

minor nit: maybe add `CMSIS-NN` to comment as it acts as kind of a title when seen in menuconfig:
```suggestion
comment "CMSIS-NN Components"
```